### PR TITLE
feat:added wishlist page

### PR DESCRIPTION
**Pull request by swethaogeti**
for preview check: https://deploy-preview-2--agitated-banach-1de3c2.netlify.app/wishlist.html

### CODE REVIEW

**Review by Debarshi95 — APPROVED** [review]
All good. Try to use semantic tags.

**Review by samyakshah3008 — APPROVED** [review]
Nice and clean code

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        </form>
+
+        <div class="nav-icons">
+            <a href="./loginpage.html" class="fas fa-user"></a>
+            <a href="#" class="fas fa-heart"></a>
+            <a href="#" class="fas fa-shopping-cart"></a>
+        </div>
+    </nav>
+
+    <div class="wishlist-container productlisting">
```
Comment (yashodaAgarwal) [inline wishlist.html:41]: Please use semantic HTML

**Review by yashodaAgarwal — APPROVED** [review]
<div class="wishlist-container productlisting">
      
        <div class="main-layout">
            <div class="container-mini">
                <h1>My Wishlist</h1>
            </div>


Use semantic code if possible

File: wishlist.html (hunk 1 above)
Comment (yashodaAgarwal) [reply wishlist.html:41]: UI looks great though

**Comment by SAGARPUJAR** [comment]
All Looks good and can be merged

**Review by SAGARPUJAR — APPROVED** [review]
UI Looks awesome and Looks good to merge

**Comment by swethaogeti** [comment]
> All good. Try to use semantic tags.

noted required changes done

File: wishlist.html (hunk 1 above)
Comment (swethaogeti) [reply wishlist.html:41]: thank u @yashodaAgarwal  suggestions are noted

**Comment by swethaogeti** [comment]
> Nice and clean code

thank you @samyakshah3008

**Pull request merged**